gym/envs/robotics/robot_env.py

```diff
@@ -83,7 +83,7 @@ def reset(self):
 
     def close(self):
         if self.viewer is not None:
-            self.viewer.finish()
+            # self.viewer.finish()
```

Comment:
new mujoco MjViewer does not have method finish(). Leaving commented and not deleted for now, but we probably need to remove it entirely.